rad/src/operators/array.rs

```diff
@@ -58,6 +58,36 @@ pub fn map(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
     Ok(RadonArray::from(result).into())
 }
 
+pub fn transpose(input: &RadonArray) -> Result<Vec<Vec<RadonTypes>>, RadError> {
+    let mut v = vec![];
+    let mut prev_len = None;
+    for item in input.value() {
+        match item {
+            RadonTypes::Array(rad_value) => {
+                let sub_value = rad_value.value();
+                let sub_value_len = sub_value.len();
+
+                if prev_len.is_none() {
+                    for sub_item in rad_value.value().into_iter() {
+                        v.push(vec![sub_item]);
```

Comment:
You could optimize this by preallocating a vector with capacity for `input.value().len()` elements, but it's a very small optimization.